src/scripts/createCsvs.ts

```diff
@@ -0,0 +1,70 @@
+import { promisify } from 'util';
+import * as fs from 'fs';
+import * as path from 'path';
+import * as Sequelize from 'sequelize';
+import { to as copyTo } from 'pg-copy-streams';
+import { Client } from 'pg';
+
+import { buildDbConfig } from '../db/models';
+
+const writeFile = promisify(fs.writeFile);
+const dbConfig = buildDbConfig();
+
+const client = new Client({
+  ...dbConfig,
+  user: dbConfig.username,
+});
+
+function main() {
+  client.connect(async err => {
+    if (err) {
+      console.error('pg cannot create client');
+      console.error(err);
+      process.exit(1);
+    }
+
+    const uploadDir = path.resolve(__dirname, '..', 'csv-exports');
```

Comment:
Only tiny tweak I might suggest is using `exports/csv` as the data dir

Reply:
done!